code/ReportAdmin.php

```diff
@@ -68,10 +68,6 @@ class ReportAdmin extends LeftAndMain implements PermissionProvider
     public function init()
     {
         parent::init();
-
-        // Set custom options for TinyMCE specific to ReportAdmin
-        HTMLEditorConfig::get('cms')->setOption('content_css', project() . '/css/editor.css');
-
```

Comment:
Not needed at best, would cause problems at worst. `app/css/editor.css` won't exist in most cases.